src/components/Header/Nav.js

```diff
@@ -3,11 +3,31 @@ import React, { Component } from 'react';
 import './Nav.css';
 
 export default class HeaderNav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: {},
+    };
+  }
+
+  componentWillMount() {
+    fetch('http://localhost:52191/Reports/Reports/GetUserData', {
```

Comment:
Should it be in the following way?

```suggestion
    return fetch('http://localhost:52191/Reports/Reports/GetUserData', {
```